services/storage-shares/pkg/config/config.go

```diff
@@ -58,3 +60,12 @@ type GRPCConfig struct {
 	Namespace string                 `yaml:"-"`
 	Protocol  string                 `yaml:"protocol" env:"STORAGE_SHARES_GRPC_PROTOCOL" desc:"The transport protocol of the GRPC service."`
 }
+
+// StatCache holds cache config
+type StatCache struct {
+	Store    string        `yaml:"store" env:"OCIS_CACHE_STORE;STORAGE_SHARES_STAT_CACHE_STORE" desc:"The type of the cache store. Supported values are: 'memory', 'ocmem', 'etcd', 'redis', 'redis-sentinel', 'nats-js', 'noop'. See the text description for details."`
```

Comment:
This adds caching to the storage-shares service - needs an addon to the service readme - you can take as reference the content from the storage-users readme as it contains a stat cache too.